compute_endpoint/tests/conftest.py

```diff
@@ -106,8 +106,8 @@ def func():
 
 
 @pytest.fixture
-def engine_heartbeat() -> float:
-    return 0.1
+def engine_heartbeat() -> int:
+    return 1
```

Comment:
Does this slow any tests down?  If so, I'd want to alter those tests to avoid the delay entirely.  0.1 seconds is an annoyance, but "tenable for now."  1 extra second per test is a problem.

Reply:
It should only slow down the tests in `test_status_reporting.py`. We can do some refactoring to reduce this in subsequent PR.